src/ripple/app/misc/impl/ValidatorList.cpp

```diff
@@ -409,15 +409,23 @@ ValidatorList::for_each_listed (
 }
 
 std::size_t
-ValidatorList::calculateQuorum (std::size_t nTrustedKeys)
+ValidatorList::calculateMinimumQuorum (
+    std::size_t nListedKeys, bool unlistedLocal)
 {
-    // Use 80% for large values of n, but have special cases for small numbers.
-    constexpr std::array<std::size_t, 10> quorum{{ 0, 1, 2, 2, 3, 3, 4, 5, 6, 7 }};
+    // Use 2f + 1 for large values of n, but have special cases for small numbers.
+    constexpr std::array<std::size_t, 6> quorum{{ 0, 1, 2, 2, 3, 3 }};
```

Comment:
Mainly because the special cases stopped being special. For 7, 8, and 9 the special quorum was the same as the new minimum. 6's special quorum is one less than the minimum, and David suggested we be conservative with numbers >5.